src/Components/News/NewsList.js

```diff
@@ -1,32 +1,20 @@
-import React from 'react';
-import '../CardListStyles.css';
-
+import React from 'react'
+import '../CardListStyles.css'
 const NewsList = () => {
-    const newsMock = [
-        {id: 2, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 1, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 3, name: 'Titulo de prueba', description: 'Descripcion de prueba'}
-    ];
-
-    return (
-        <div>
-            <h1>Listado de Novedades</h1>
-            <ul className="list-container">
-                {newsMock.length > 0 ? 
-                    newsMock.map((element) => {
-                        return(
-                            <li className="card-info" key={element.id}>
-                                <h3>{element.name}</h3>
-                                <p>{element.description}</p>
-                            </li>
-                        )
-                    })
-                :
-                    <p>No hay novedades</p>
-                }
-            </ul>
-        </div>
-    );
+  const cardsFutureComponent = [{ id: 2 }, { id: 1 }, { id: 3 }]
+  return (
+    <div>
+      <h1>Novedades</h1>
+      <ul className='list-container'>
+        {cardsFutureComponent.length > 0 ? (
```

Comment:
Maybe in the fututure you can add a function listHasValues on Utils/validations file